### DISCUSSION TITLE
Dashboard "scratch" line at top and bottom

### DISCUSSION THREAD

**Discussion by suzhiking** [folke/snacks.nvim#109] [Q&A]
To hide the lualine and bufferline, I set their startup event to "BufReadPre", however, the dashboard then have two "scratch" line at both top and bottom. Anyone know how to resolve this?

<img width="1905" alt="Screenshot 2024-11-19 at 1 34 06 AM" src="https://github.com/user-attachments/assets/cd64476b-6ad3-4773-a7d7-0684c7172e19">

**Comment by folke** [comment]
Just pushed an update that hides this automatically

**Reply by suzhiking** [reply]
You are the GOAT.

**Reply by suzhiking** [reply]
Also I found that in the function

```lua
-- Adds a section to restore the session if any of the supported plugins are installed.
---@param item? snacks.dashboard.Item
---@return snacks.dashboard.Item?
function M.sections.session(item)
  local plugins = {
    ["persistence.nvim"] = ":lua require('persistence').load()",
    ["persisted.nvim"] = ":SessionLoad",
    ["neovim-session-manager"] = ":SessionManager load_current_dir_session",
    ["possession.nvim"] = ":PossessionLoadCwd",
    ["mini.sessions"] = ":lua require('mini.sessions').read('local')",
    ["mini.nvim"] = ":lua require('mini.sessions').read('local')",
  }
  for name, action in pairs(plugins) do
    if M.have_plugin(name) then
      return setmetatable({ -- add the action and disable the section
        action = action,
        section = false,
      }, { __index = item })
    end
  end
end
```
when mini.nvim is installed but mini.session is not enabled, while I'm using persistence.nvim, there is 50% chance that this error occurred: 

<img width="1900" alt="Screenshot 2024-11-19 at 2 31 45 AM" src="https://github.com/user-attachments/assets/9d3c111b-a0e2-4bd3-8ab3-09466e4696f4">

and it could be fixed by removing entry
```lua
["mini.nvim"] = ":lua require('mini.sessions').read('local')",
```
Do you know any other workarounds with this?

**Reply by folke** [reply]
Fixed as well
